--- fstab	2016-07-21 14:43:08.000000000 -0700
+++ fstab.new	2016-07-21 14:43:13.000000000 -0700
@@ -1,4 +1,5 @@
 proc            /proc           proc    defaults          0       0
 /dev/mmcblk0p1  /boot           vfat    defaults          0       2
 /dev/mmcblk0p2  /               ext4    defaults,noatime  0       1
+/dev/mmcblk0p3  /data           ext4    defaults,noexec,noatime  0       1
 tmpfs		/run/shm	tmpfs	defaults,noexec,nosuid	0	0
